src/config/passport.js

```diff
@@ -1,26 +1,27 @@
-const passport = require("passport");
-const LocalStrategy = require("passport-local").Strategy;
-const mongoose = require("mongoose");
-const User = mongoose.model("User");
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const mongoose = require('mongoose');
+
+const User = mongoose.model('User');
 
 passport.use(
-    new LocalStrategy(
-        {
-            usernameField: "user[email]",
-            passwordField: "user[password]"
-        },
-        function(email, password, done) {
-            User.findOne({ email: email })
-                .then(function(user) {
-                    if (!user || !user.validPassword(password)) {
-                        return done(null, false, {
-                            errors: { "email or password": "is invalid" }
-                        });
-                    }
+  new LocalStrategy(
+    {
+      usernameField: 'user[email]',
+      passwordField: 'user[password]'
+    },
+    ((email, password, done) => {
+      User.findOne({ email })
+        .then((user) => {
+          if (!user || !user.validPassword(password)) {
+            return done(null, false, {
+              errors: { 'email or password': 'is invalid' }
+            });
+          }
 
-                    return done(null, user);
-                })
-                .catch(done);
-        }
-    )
+          return done(null, user);
+        })
+        .catch(done);
+    })
+  )
 );
```

Comment:
do we need this file, if yes why?

Reply:
I actually don't need it since I  didn't use passport